content/commons.json

```diff
@@ -58,9 +59,14 @@
     "not-found:subtitle": "Page not found",
     "not-found:button": "Go home",
     
+    "navigation:news": "News",
     "navigation:schedule": "Schedule",
     "navigation:library": "Library",
 
+    "news:title": "News",
+    "news:description": "This is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.",
+
```

Comment:
Placeholder text that needs to be replaced with the actual description for the page. The title may be changed.